libs/feature/search/src/lib/utils/service/fields.service.ts

```diff
@@ -39,6 +39,9 @@ marker('search.filters.publisherOrg')
 marker('search.filters.user')
 marker('search.filters.changeDate')
 
+marker('search.filters.summaryLabel.user')
+marker('search.filters.summaryLabel.changeDate')
```

Comment:
how about marking labels for all fields? It might make more sense taken from outside I think. Another approach would be to use the field normal label if a summaryLabel is not available.

Reply:
Finally, I've added a fallback  in [6b01ea9](https://github.com/geonetwork/geonetwork-ui/pull/1037/commits/6b01ea922a2955741921e530aa92146664274d09) to use the filters label, as we don't have the exact wording for more filter summary labels yet.